packages/nimbus/src/components/date-range-picker/date-range-picker.mdx

```diff
@@ -12,82 +13,206 @@ tags:
   - component
```

Comment:
Just as an aside, we should probably start doing more with tags.
`date`, `calendar`, etc come to mind